syntax = "proto3";

package fraud_detection;

service FraudDetection {
    rpc LookforFraud (FraudRequest) returns (FraudResponse);
}

message UserData {
    string name = 1;
    string contact = 2;
}

message CreditCard {
    string number = 1;
    string expirationDate = 2;
    string cvv = 3;
}

message FraudRequest {
    UserData user = 1;
    CreditCard cc = 2;
}


message FraudResponse {
    bool fraud = 1;
}
